application/src/js/tablebuilder2/tablebuilder2.js

```diff
@@ -136,14 +137,12 @@ $(document).ready(function () {
                 populateEthnicityPresets(presets);
 
                 // show the presets (step 2) and table type (step 3) section
-                $('#ethnicity_settings_section').show();
+                document.getElementById('ethnicity_settings_section').classList.remove('hidden')
 
                 if (getIsSimpleData(table_data)) {
-                    $('#simple_table_options').show();
```

Comment:
Have we lost the remove('hidden') for `#simple_table_options`?

Reply:
@samuelhwilliams that was deliberate too, I think, as there are no options for simple tables...